sesame/src/main/scala/org/w3/banana/sesame/SesameModule.scala

```diff
@@ -38,6 +38,8 @@ trait SesameModule
 
   implicit val rdfStore: RDFStore[Sesame, Try, RepositoryConnection] with SparqlUpdate[Sesame, Try, RepositoryConnection] = new SesameStore
 
+  implicit val transactor: Transactor[Sesame, RepositoryConnection] = rdfStore
```

Comment:
This does not feel right - do really need the transactor as a separate trait when the store itself already includes it? Also, this then differs from the Jena Implementation so that should be changed. Again, I would really rather have this discussed on a separate issue first so that it is easier to focus on the main point of this PR.

Coalescing the transactional semantics of different data stores is enough for one PR: